swap_meet/vendor.py

```diff
@@ -0,0 +1,96 @@
+from swap_meet.item import Item
+
+#Wave 1 
+class Vendor:   
+    def __init__(self, inventory = None):
+        if inventory == None:
+            self.inventory = []
+        else: 
+            self.inventory = inventory
+    
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
Consider using the pattern where you check the edge case first:

```suggestion
        if item not in self.inventory:
            return False
            
        self.inventory.remove(item)
        return item
```